src/puyapy-stubs/_contract.pyi

```diff
@@ -3,6 +3,28 @@ import typing
 
 from puyapy import UInt64, urange
 
+class StateTotals:
+    """
+    Defines the total amount of state per resource and type a contract will use.
+
+    This is not required when using the declarative state proxies, but may be required if a
+    contract dynamically interacts with state via AppGlobal.get_bytes etc
+    """
+
+    def __init__(
+        self,
+        *,
+        global_uints: int | None = None,
```

Comment:
I think rather than allowing a None can we use `...` as the default value?
e.g. `global_uints: int = ...`

Reply:
For expediency resolved in a commit myself 🙂